wpilibc/src/main/native/include/frc/ADIS16470_IMU.h

```diff
@@ -425,6 +442,7 @@ class ADIS16470_IMU : public wpi::Sendable,
   /** @brief Resources **/
   DigitalInput* m_reset_in = nullptr;
   DigitalOutput* m_status_led = nullptr;
+  Rotation3d angleOffset = Rotation3d{0_deg, 0_deg, 0_deg};
```

Comment:
This should be removed for same reasons as the one in `ADIS16448_IMU.h` (see https://github.com/wpilibsuite/allwpilib/pull/6290#discussion_r1585311865).